app/assets/javascripts/discourse/app/templates/components/sidebar/section-link.hbs

```diff
@@ -7,14 +7,39 @@
     @current-when={{@currentWhen}}
     @title={{@title}}
   >
+    {{#if @prefixValue }}
+      <span class="sidebar-section-link-prefix {{@prefixType}} {{@prefixCSSClass}}" style={{this.prefixCSS}}>
+        {{#if (eq @prefixType "image")}}
+          <img src={{@prefixValue}}>
+        {{/if}}
+        {{#if (eq @prefixType "text")}}
+          {{@prefixValue}}
+        {{/if}}
+        {{#if (eq @prefixType "icon")}}
+          {{d-icon @prefixValue }}
+        {{/if}}
+        {{#if @prefixBadge}}
+          {{d-icon @prefixBadge class="prefix-badge"}}
+        {{/if}}
+      </span>
+    {{/if}}
     <span class="sidebar-section-link-content-text">
       {{@content}}
     </span>
 
+    {{! TODO move badge count to suffixType="text", suffixValue="3" like prefix}}
     {{#if @badgeCount}}
       <span class="sidebar-section-link-content-badge">
         {{@badgeCount}}
       </span>
     {{/if}}
+
+    {{#if @suffixValue }}
+      <span class="sidebar-section-link-suffix {{@suffixType}} {{@suffixCSSClass}}">
```

Comment:
Same concern as https://github.com/discourse/discourse/pull/17296/files#r920811904

Reply:
In suffix similarly, I am using class for new messages indicator. Blue for just new messages and green for mentions.

Reply:
Got it :+1: